docs/source/index.rst

```diff
@@ -1,8 +1,21 @@
-Welcome to astrolabe's documentation!
-=====================================
+Drivers Atlas Testing
+=====================
 
-**astrolabe** is a distribution containing tools for automating Atlas operations and running Atlas Planned Maintenance
-Tests.
+The Drivers Atlas Testing project validates the behavior of drivers against
```

Comment:
Let's not repeat text in our docs? Its a nightmare to maintain. I suggest keeping the first paragraph here and then linking to the architecture guide. The corresponding paragraph can be removed from the architecture page entirely.